### PR TITLE
feat(client): support edgeExistence api

### PR DESCRIPTION
**Pull request by msgui**
## Purpose of the PR

- follow https://github.com/apache/incubator-hugegraph/pull/2242

## Main Changes
- support edgeexistence api

## Verifying these changes

- [ ] Trivial rework / code cleanup without any test coverage. (No Need)
- [ ] Already covered by existing tests, such as *(please modify tests here)*.
- [ ] Need tests and can be verified as follows:
    - xxx

## Does this PR potentially affect the following parts?

- [x]  Nope
- [ ]  Dependencies (add/update license info) 
- [ ]  Modify configurations
- [ ]  The public API
- [ ]  Other affects (typed here)

## Documentation Status

- [ ]  `Doc - TODO` 
- [ ]  `Doc - Done` 
- [ ]  `Doc - No Need`

### CODE REVIEW

**Comment by msgui** [comment]
I've run into a challenging issue, seemingly caused by errors related to the COMMIT_ID, resulting in GitHub CI not running smoothly. 
`fatal: reference is not a tree: be6ee386b9939dc6bd6fcbdf2274b8acc3a0a314`


image

**Comment by imbajin** [comment]
> I've run into a challenging error, seemingly caused by errors related to the **'COMMIT_ID'**, resulting in GitHub CI not running smoothly. `fatal: reference is not a tree: be6ee386b9939dc6bd6fcbdf2274b8acc3a0a314`
> 
> image

refer https://github.com/apache/incubator-hugegraph-toolchain/pull/542#issuecomment-1833504699

File: hugegraph-client/src/main/java/org/apache/hugegraph/api/traverser/EdgeExistenceAPI.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+    @Override
+    protected String type() {
+        return "edgeexist";
+    }
+
+    public List<Edge> get(Object sourceId, Object targetId, String edgeLabel,
+                          String sortValues, int limit) {
+        String source = GraphAPI.formatVertexId(sourceId, false);
+        String target = GraphAPI.formatVertexId(targetId, false);
```
Comment (msgui) [inline hugegraph-client/src/main/java/org/apache/hugegraph/api/traverser/EdgeExistenceAPI.java:45]: The id obtained through GraphAPI.formatVertexId may contain unnecessary double quotation marks.

8273a2dadac8f6b43e742d90a366c87

**Comment by msgui** [comment]
https://github.com/apache/incubator-hugegraph-toolchain/pull/544 After merging, continue.

**Comment by imbajin** [comment]
merge after #550

**Review by imbajin — APPROVED** [review]

**Review by coderzc — APPROVED** [review]

**Pull request merged**